deployer/templates/heapster.yaml

```diff
@@ -66,7 +66,6 @@ objects:
           - "--source=kubernetes:${MASTER_URL}?useServiceAccount=true&kubeletHttps=true&kubeletPort=10250"
           - "--sink=hawkular:https://hawkular-metrics:443?tenant=_system&labelToTenant=pod_namespace&caCert=/hawkular-cert/hawkular-metrics-ca.certificate&user=%username%&pass=%password%&filter=label(container_name:^/system.slice.*|^/user.slice)"
           - "--logtostderr=true"
-          - "--vmodule=*=4"
```

Comment:
Is this a value that can be moved to an env var for the deployment so it can be updated and verbose logs can be enabled?